LovelyCrane/Common/Assets.swift

```diff
@@ -7,10 +7,20 @@
 
 import SwiftUI
 
+enum Crans: String, CaseIterable {
```

Comment:
```suggestion
enum Cran: String, CaseIterable {
```

s가 혹시 복수의 의미를 나타내는거라면 배열이 아닌, enum이기 때문에 없어도 될 것 같아요!

Reply:
넹!